### PR TITLE
Bump v1.5.1 to v1.5.2 and update changelog

### PR DESCRIPTION
**Pull request by langner**
~~DO NOT SUBMIT~~

~~We will merge this in when we are ready with all the issues assigned to v1.5.2 are either closed or moved over to the next milestone.~~

This should be good to go.

### CODE REVIEW

File: ANNOUNCE [hunk 1]
```diff
@@ -37,7 +37,7 @@ The cclib development team
 ———
 
 Summary of changes since last version:
-    New attribute polarizabilities for static or dynamic dipole polarizability
-    New attribute pressure for thermochemistry
-    Detecting closed shells in parsed data
-    Support for Python 3.6
+    * Support for writing Molden and WFX files
+    * Support for thermochesmitry attributes in ORCA
+    * Support for chelpg atomic charges in ORCA (Richard Gowers)
```
Comment (TDI-tenderholt) [inline ANNOUNCE]: Do you want to make this more consistent with the other points? Suggest either removing (Richard Gowers) or adding the other contributors.

File: CHANGELOG [hunk 2]
```diff
@@ -1,3 +1,20 @@
+Changes in cclib-1.5.2:
+
+Features:
+    * Support for writing Molden and WFX files (Gaur Sagar)
```
Comment (TDI-tenderholt) [inline CHANGELOG]: Shouldn't it be Sagar Gaur?

File: ANNOUNCE (hunk 1 above)
Comment (langner) [reply ANNOUNCE]: Removed.

File: CHANGELOG (hunk 2 above)
Comment (langner) [reply CHANGELOG]: Done.

**Pull request merged**